.github/workflows/release_npm_version.yml

```diff
@@ -63,4 +63,31 @@ jobs:
                 console.log(`Release created`);
             }
             createRelease();
-            
+  triggerRNSandboxProd:
+    name: Trigger React Native Prod Sandbox release
+    runs-on: ubuntu-latest
+    needs: createRelease
+    steps:
+      - name: Trigger ReactNative Sandbox Prod Deployment
+        run: |
+         curl \
+             -X POST \
+             -H "Accept: application/vnd.github.v3+json" \
+             -H "Authorization: token ${{ secrets.GPR_API_KEY }}" \
+             https://api.github.com/repos/Neuro-ID/neuroid-reactnative-sdk-sandbox/dispatches \
+             -d '{"event_type":"publish-prod-android","client_payload":{"version":"${{env.PACKAGE_VERSION}}", "message": "${{ github.event.commit.message }}"}}'
```

Comment:
This just covers the Android PROD release, should we add the iOS Trigger?

Reply:
will create a separate ticket for the iOS RN workflow.